src/pages/editor/EditorInstitution.tsx

```diff
@@ -33,6 +38,23 @@ export const EditorInstitution = () => {
     meta: { errorMessage: t('feedback.error.get_institution') },
   });
 
+  const institutionUsersQuery = useQuery({
+    queryKey: ['institutionUsers', customerId],
+    enabled: !!customerId,
+    queryFn: () => (customerId ? fetchUsers(customerId, [RoleName.Editor, RoleName.InstitutionAdmin]) : undefined),
+    meta: { errorMessage: t('feedback.error.get_users_for_institution') },
+  });
+
+  const institutionUsers = institutionUsersQuery.data ?? [];
+
+  const institutionAdmins = institutionUsers.filter((user) =>
+    user.roles.some((role) => role.rolename === RoleName.InstitutionAdmin)
```

Comment:
```suggestion
    user.roles.includes(RoleName.InstitutionAdmin)
```

Reply:
Roles er ikke et array av strenger, men av objekter. Bruker derfor `some()` for conditional checking.

Reply:
Ah, såklart, my bad 👍